.dockerignore

```diff
@@ -0,0 +1,8 @@
+models/
```

Comment:
You will have to add a license/copyright header here

Reply:
Good catch, added